Formula/infrakit.rb

```diff
@@ -0,0 +1,30 @@
+class Infrakit < Formula
+  desc "Toolkit for creating and managing declarative infrastructure"
+  homepage "https://github.com/docker/infrakit"
+  url "https://github.com/docker/infrakit.git",
+      :tag => "v0.5",
+      :revision => "3d2670e484176ce474d4b3d171994ceea7054c02"
+  head "https://github.com/docker/infrakit.git"
```

Comment:
Kill this line please.

Reply:
Is `--HEAD` no longer supported?? I always thought that it was one of the nicest feature of homebrew.

Reply:
It's supported for formulae that have existed for a while if there proves enough demand for it, essentially.

Reply:
Yea makes sense, it also reduces maintenance overhead for not having it in all formulas but it does help test new versions of a program.